python-package/lightgbm/plotting.py

```diff
@@ -603,6 +611,7 @@ def create_tree_digraph(
     precision: Optional[int] = 3,
     orientation: str = 'horizontal',
     example_case: Optional[Union[np.ndarray, pd_DataFrame]] = None,
+    max_category_values: Optional[int] = None,
```

Comment:
Instead of defaulting to `None` and needing to deal with the complexity of `if max_category_values is None` conditions throughout the code, in my opinion it would be preferable to just choose a high default value like `50` or `100` or whatever you think could reasonably be expected to display well by default.

@jmoralez what do you think?

Reply:
I agree with that, @moziada can you please make that change? The type of max_category_values would be an int with the default value that you choose.

Reply:
I have defaulted ```max_category_values``` to ```32``` as ```max_cat_threshold``` [parameter](https://lightgbm.readthedocs.io/en/latest/Parameters.html#max_cat_threshold). What do you think?

Reply:
I think your original example in #5687 has around 32 values
![image](https://user-images.githubusercontent.com/8473587/231557584-20f2107d-862f-4ec7-903b-f0afa3e51829.png)
the `max_cat_threshold` refers to bins I think, so for example bin 0 could be 1||3, bin 1 2||5, etc.

 Maybe around 10 would be better. WDYT?

Reply:
yes on the example it had 32 values, I made a commit changing the default value to 10